src/isar_exr/api/energy_robotics_api.py

```diff
@@ -188,12 +188,14 @@ def create_waypoint_task_definition(
             raise RobotException(e)
 
         return response_dict["createWaypointTaskDefinition"]["id"]
-    
-    def wake_up_robot(self, exr_robot_id: str) -> None:
+
+    def wake_up_robot(
+        self, exr_robot_id: str, startTime: datetime = datetime.today()
```

Comment:
Not sure it makes sense to pass startTime as an input to this function?
What does it mean to the functionality of "wake_up_robot" ?

Reply:
This is done to make the function testable. Or else, we have to wait the full MAX_TIME_FOR_WAKEUP to test whether or not the function throws an exeption at timeout

Reply:
Couldn't we instead change the settings for the test?